frontend/awx/main/AwxLogin.tsx

```diff
@@ -37,16 +36,15 @@ export function AwxLogin(props: { children: React.ReactNode }) {
 
   if (!activeAwxUser) {
     return (
-      <Login
+      <AnsibleLogin
         authOptions={authOptions}
-        apiUrl="/api/login/"
+        loginApiUrl="/api/login/"
```

Comment:
Interesting that the other projects need to have the functions to wrap those URLS.

Reply:
In the AWX API the login is not under the /api/v2 path so we cannot wrap it.
Since that login is not used downstream, this works .